File: check_fajr_corners.js
const Jimp = require('jimp');

Jimp.read('fajr_extracted_1.png').then(image => {
  const w = image.bitmap.width;
  const h = image.bitmap.height;
  
  const corners = [
    {x: 0, y: 0},
    {x: w-1, y: 0},
    {x: 0, y: h-1},
    {x: w-1, y: h-1},
  ];
  
  console.log(`Dimensions: ${w}x${h}`);
  corners.forEach((c, i) => {
    const hex = image.getPixelColor(c.x, c.y).toString(16).padStart(8, '0');
    console.log(`Corner ${i}: ${hex}`);
  });
}).catch(console.error);

File: check_fajr_corners.py
from PIL import Image

img = Image.open('fajr_extracted_1.png').convert('RGBA')
w, h = img.size

corners = [
    (0, 0),
    (w-1, 0),
    (0, h-1),
    (w-1, h-1)
]

print(f"Dimensions: {w}x{h}")
for i, c in enumerate(corners):
    r,g,b,a = img.getpixel(c)
    print(f"Corner {i}: R={r} G={g} B={b} A={a}")
